### DISCUSSION TITLE
How to use Earth inOrange3 ?

### DISCUSSION THREAD

**Discussion by pgr123** [biolab/orange3#6666] [Q&A]
I try to use Earth with Orange3 but I get errors when I try examples : 
import Orange
data = Orange.data.Table("housing")
c = Orange.regression.earth.EarthLearner(data, degree=2, terms=10)
print(c)

I got that error : 
  File "/home/pascal/Bureau/toto2.py", line 3, in <module>
    c = Orange.regression.earth.EarthLearner(data, degree=2, terms=10)
  File "/home/pascal/.local/lib/python3.10/site-packages/Orange/misc/lazy_module.py", line 13, in __getattr__
    return getattr(self._do_import(), key)
AttributeError: module 'Orange.regression' has no attribute 'earth'

Please, help me.
I use Linux Mint 21.2 with Python 3.10.12, numpy and scipy, ...
Thank you.

Pascal

**Comment by markotoplak** [comment]
I think this was only an Orange2 add-on and it is not available for Orange3. @ales-erjavec, is this correct?

**Comment by ales-erjavec** [comment]
>>> I think this was only an Orange2 add-on and it is not available for Orange3. @ales-erjavec, is this correct?

Yes.

**Comment by pgr123** [comment]
It's a shame, so there's no way to use Earth with Python 3.10 ! What a pity ! It's too bad !
